### ISSUE TITLE
Default expiry?

### ISSUE THREAD

**Issue by sumBTC** [ACINQ/eclair#1426]
I get the **impression** that the default expiry of Eclair is 6 hours (21600 seconds) and for Phoenix it's 1 hour (3600 seconds). 

Payment requests have an expiry, please make explicit in the main documentation what the default expiry is and if it's adjustable (**edit** without having to install the wallet) Thanks!

**Comment by t-bast** [comment]
What makes you say that? The default is 1 hour everywhere: https://github.com/ACINQ/eclair/blob/bfd25fa0211d7161157541f3ee7fb67aa6282888/eclair-core/src/main/resources/reference.conf#L112

As explained in our README, you simply need to look into this file for all the configuration default values, and everything can be changed.

**Comment by sumBTC** [comment]
Ah, thanks. I looked in the "Search or jump to..." bar for different values in seconds, especially 3600. I should have searched for "hour" there. When I do the same for Phoenix, 3600 comes up with nothing and hour also. I was especially interested in what the App users get as default value and whether it's adjustable (without necessarily installing the App myself). For Phoenix (that I have installed, Eclair not) I had to look inside the invoice to find out what the expiry was. For Eclair I asked a Reddit user who explained it to me: "Default is 1h. It can be adjusted down to 20m and up to 6h."
